src/reference/build.gradle

```diff
@@ -0,0 +1,72 @@
+buildscript {
+  dependencies {
+    classpath 'org.asciidoctor:asciidoctorj-epub3:1.5.0-alpha.6'
+    classpath 'org.asciidoctor:asciidoctorj-pdf:1.5.0-alpha.15'
+  }
+}
+
+plugins {
+  id 'de.undercouch.download' version '3.4.3'
+  id 'org.asciidoctor.convert' version '1.5.3'
+}
+
+apply plugin: 'java'
+apply plugin: 'org.asciidoctor.convert'
+
+version = '1.0.0-SNAPSHOT'
+
+asciidoctorj {
+  version = '1.5.5'
+}
+
+task downloadResources(type: Download) {
+  src 'http://repo.spring.io/libs-snapshot-local/org/springframework/doc-resources/spring-doc-resources/0.1.1-SNAPSHOT/spring-doc-resources-0.1.1-SNAPSHOT.zip'
```

Comment:
What is this about?
Can't we at least extract a variable for the version of this artifact?

Reply:
Putting it in a variable would just add another line.